index.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <header class="container">
+
+        <nav class="toolbar">
+            <div class="logo"><a class="logo-txt" href="#">My Logo</a></div>
+            <div class="menu-image"><img src="./images/ic_menu.jpg" alt="menu"></div>
+        </nav>
+
+        <div class="circle">
+            <br id="rec">
+
+        </div>
```

Comment:
- [x] [ OPTIONAL ] Kindly make sure the container of this section is a `header `tag. 
Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. :grinning:  :grinning:  :grinning: 

- [x] [ OPTIONAL ] The logo (if the design has one) is on the left and has a link (`a` tag).
   * This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.  :grinning:  :grinning:  :grinning: 

- [x] [ OPTIONAL ] The hamburger button is on the right and has a `nav` tag.
   * The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.  :grinning:  :grinning:  :grinning: 

 - check the example below :+1: 

```
<header>
        <a href="index.html">
          <img src="images/logo.png" alt="Logo" />
        </a>

        <nav class="hamburger-menu">
          <div class="bar-top"></div>
          <div class="bar-middle"></div>
          <div class="bar-bottom"></div>
        </nav>
    </header>
```